### DISCUSSION TITLE
Lag en footer for prosjektene som inneholder versjon og nyttige lenker

### DISCUSSION THREAD

**Discussion by olemp** [Puzzlepart/prosjektportalen365#1026] [Ideas]
Hard to avoid issues with this as long as we're stuffing it into the hub menu. I suggest creating a PP-specific footer instead. Might be other use cases for this as well down the line (e.g. "PP365 on GitHub", "Report an issue", "Suggest xyz").

![image](https://user-images.githubusercontent.com/58074707/203545019-734f3a17-33fa-4f3c-a49a-7aaa0108ca18.png)

_Originally posted by @tarjeieo in https://github.com/Puzzlepart/prosjektportalen365/issues/902#issuecomment-1353592022_

**Comment by Remi749** [comment]
The "Hjelp tilgjengelig" button is also hidden if users use "Elements" plugin. Example:

![image](https://user-images.githubusercontent.com/28678468/210755009-7c0eaa2f-3ae8-40b5-8e5a-ad9453cfbb32.png)

The root element is hidden.
![image](https://user-images.githubusercontent.com/28678468/210755303-8298618b-6b78-4e89-9054-5c91bbc44295.png)

**Comment by Remi749** [comment]
Quick fix for this is to set a height for the "pp-help-content" div. This will put "? Help tilgjengelig" button on a line above the top navigation. This can also be done responsive if menu items clash with the button. 

Do we really want to put this in a footer at the bottom? I think it will hide the help content feature too much and be overlooked. 

I agree on all the other stuff that has been discussed that should go in a footer. @tarjeieo, thoughts?

**Comment by Remi749** [comment]
Before: 
![image](https://user-images.githubusercontent.com/28678468/210758609-f083ee00-9a1c-4750-8aa6-efd5f20c4a24.png)

After:
![image](https://user-images.githubusercontent.com/28678468/210758659-4092ccdd-de48-4e91-883a-021a71ddd2e3.png)

**Comment by Remi749** [comment]
If customer has "Elements" plugin, it will look like this:
![image](https://user-images.githubusercontent.com/28678468/210758785-5a9272c4-412a-4d07-ad17-657cd17a214b.png)

With the quick fix it will not be hidden. But there will be some unnessecary whitespace for those that uses "Elements"

**Comment by tarjeieo** [comment]
Kindof agree with you, although I think a footer have merits in the long run. I suggest you open https://github.com/Puzzlepart/prosjektportalen365/issues/902 again and fix it by your quick fix - then we'll do a more thorough discussion on the footer later this year.

**Comment by olemp** [comment]
Her må vi brainstorme litt og tenke på innhold til footeren, derav konvertering til diskusjon.

**Comment by tarjeieo** [comment]
Footer v1
- Versjonsnummer og evt kanal
- Hjelp tilgjengelig
- Lenker fra sentral liste

Sentral lenkeliste på hub

Må teste footeren i kombinasjon med global footer-options

**Reply by olemp** [reply]
Hei @okms, @tarjeieo, @Remi749 og @pzljanb 

En minimal versjon ([v0.1](https://github.com/Puzzlepart/prosjektportalen365/issues/1070)) er nå tilgjengelig i [testkanalen](https://puzzlepart.sharepoint.com/sites/pp365_channels_test). 

### Lenke til "Konfigurasjon av Prosjektportalen"
![image](https://user-images.githubusercontent.com/7606007/228861081-a7b01e09-42e6-499f-8d3f-3c09e18c5879.png)

### Installasjonsinformasjon 
![image](https://user-images.githubusercontent.com/7606007/228861138-4e6deeb8-0202-4877-8988-4ec1607177b5.png)

### Hvordan man ligger an i forhold til siste versjon på GitHub
![image](https://user-images.githubusercontent.com/7606007/228861188-85bc64db-7645-48f5-ae97-eae895bb8679.png)

Om det finnes en nyere versjon på GitHub kan man laste den ned direkte:
![image](https://user-images.githubusercontent.com/7606007/228861714-a8c409a8-b215-40ed-a0ee-68f6586e31bf.png)
